libraries/platforms/dromajo-vcs/link.mk

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2019, University of Washington All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification,
+# are permitted provided that the following conditions are met:
```

Comment:
Note for later: Need to think about using the new VCS compilation method, but that can be deferred to later.